scripts/cloud-check-control/cloudAtlasCheckControl.py

```diff
@@ -700,10 +711,14 @@ def parse_args():
         required=True,
         help="JAR - s3://path/to/atlas_checks.jar or /local/path/to/atlas_checks.jar to execute",
     )
+    parser_check.add_argument(
+        "--db",
+        help="db - The full path to the db folder used for external db checks.",
```

Comment:
Since this is more "extra data files", maybe `--db` -> `--extra`, and `help="extra - The full path to the folder used for extra data (for example elevation data and databases)"`.

Reply:
Done